benchbuild/cli/project.py

```diff
@@ -33,6 +35,40 @@ def main(self, *projects: str) -> int:
         return 0
 
 
+@BBProject.subcommand("details")
+class BBProjectDetails(cli.Application):
+    """Show details for a project."""
+
+    def main(self, project: str) -> int:
+        index = bb.populate([project], [])
+        if not index.values():
+            print(f'Project named {project} not found in the registry.')
+            print('Maybe it is not configured to be loaded.')
+            return -1
+        for project_cls in index.values():
+            print_project(project_cls)
+        return 0
+
+
+def print_project(project: tp.Type[Project]) -> None:
+    print(f'project: {project.NAME}')
+    print('source:')
+    for source in project.SOURCE:
+        print(' ', f'{source.remote}')
+        print('versions')
+        for v in source.versions():
+            print(' ', v)
```

Comment:
This can get very long for large git repos, doesn't it?
Maybe only print the first `n` and add `...` if there are more?

Reply:
It is limited to 10 and can be expanded. After our discussion today, I added a reference to the 'local' directory for the user.